packages/p2p/src/components/buy-sell/buy-sell-row.scss

```diff
@@ -90,6 +90,14 @@
 
     &__button {
         align-self: flex-start;
+        display: flex;
+        flex-flow: row wrap;
+        white-space: inherit;
+        max-width: 13rem;
```

Comment:
Added `max-width: 13rem` by design requirements after consulting with Yunus, after the button expands within a certain fixed width, wrap the text and make the button size cater to the overflown text